src/rust/engine/fs/store/src/lib.rs

```diff
@@ -1997,25 +2125,46 @@ mod remote {
     pub fn list_missing_digests(
```

Comment:
A general thought here is that it would be great to contribute some sort of tracing support upstream in the RPC framework that we are using, and to then hook into it.

This relates a bit to #7256, because tower gRPC is very likely to support https://github.com/tokio-rs/tracing, and if we were to add support to `tracing` for recording to our workunit store, we would get this "for free" without individually annotating our RPCs (because the underlying RPC framework would be annotating "a gRPC method invoke" more generically).

@illicitonion : Food for thought?